client/ayon_core/pipeline/create/creator_plugins.py

```diff
@@ -150,15 +156,15 @@ def remove_convertor_item(self):
 class BaseCreator:
     """Plugin that create and modify instance data before publishing process.
 
-    We should maybe find better name as creation is only one part of it's logic
+    We should maybe find better name as creation is only one part of its logic
     and to avoid expectations that it is the same as `avalon.api.Creator`.
 
     Single object should be used for multiple instances instead of single
     instance per one creator object. Do not store temp data or mid-process data
     to `self` if it's not Plugin specific.
 
     Args:
-        project_settings (Dict[str, Any]): Project settings.
+        project_settings (dict[str, Any]): Project settings.
```

Comment:
I've actually started to use `Dict` and `List` so they can be imported and used without `""` (in the code).

Reply:
Sure - I understand, I ended up doing that only for those that are in the code and not in strings like docstrings. Also because `List` and `Dict` as far as I know are actually deprecated already 🤦 

Would you like me to stick with `List` and `Dict` for now?

Also note that the linter we have configured for Github doesn't recognize an import from `typing` if it's only used in str or docstring?

Reply:
> Also because List and Dict as far as I know are actually deprecated already 🤦

Omg, why we always move to current end of life? Then keep it as it is...